src/commands/load.js

```diff
@@ -0,0 +1,33 @@
+let path = require('path');
+
+module.exports = () => {
+	return {
+		run: function(args, api, event) {
```

Comment:
About that spacing - I have my Notepad++ set to make 4 space intendation. These tab spacing was there already, I copied that code fragment you quoted here from reload.js. I can correct that of course alone with other issues, will do that tomorrow.